src/gsp.py

```diff
@@ -79,10 +87,15 @@ def get_all_available_forecasts(
     - **start_datetime_utc**: optional start datetime for the query. e.g '2023-08-12 10:00:00+00:00'
     - **end_datetime_utc**: optional end datetime for the query. e.g '2023-08-12 14:00:00+00:00'
     """
+    if is_fake:
+        if gsp_ids is None:
+            gsp_ids = [int(gsp_id) for gsp_id in range(GSP_TOTAL)]
+
+        make_fake_forecasts(gsp_ids=gsp_ids, session=session)
 
     logger.info(f"Get forecasts for all gsps. The option is {historic=} for user {user}")
 
-    if gsp_ids is not None:
+    if isinstance(gsp_ids, str):
```

Comment:
this might have caused one of the tests to failed

Reply:
Oh, that's weird. I don't have my computer on right now, but I can dig into this tomorrow, if that's cool?

Reply:
I think it needs to happen before the `is_fake` thing

Yea no problem

Reply:
I've moved the isinstance() check to occur prior to the is_fake() condition. Could trigger the docker file again to run the test, please?

Reply:
I've moved the `isinstance()` check to occur prior to the `is_fake()` condition. Could you trigger the docker file again to run the test, please?